### PR TITLE
[FE-17] [FE-18] Init group table

### PR DESCRIPTION
**Pull request by AnCichocka**
- create group table without edit & awards
- add category filters

https://oops2000.atlassian.net/browse/SCRUM-17
https://oops2000.atlassian.net/browse/SCRUM-18

image

### CODE REVIEW

File: frontend/src/hooks/Group/useGroupScreenData.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        student: {
+          id: user?.userId ?? "-1",
+          fullName: `${user?.firstName ?? "-"} ${user?.secondName ?? "-"}`,
+          index: user?.indexNumber ?? -1,
+        },
+        subcategories:
+          userPoints?.categoriesPoints.flatMap((catPoints) =>
+            catPoints.subcategoryPoints.map((subPoints) => {
+              return {
+                pure: subPoints.points.purePoints?.value ?? 0,
```
Comment (Szyntos) [inline frontend/src/hooks/Group/useGroupScreenData.tsx:67]: Should this be
```js
 pure: subPoints.points.purePoints?.value ?? "-",
```
?

**Review by Szyntos — CHANGES_REQUESTED** [review]
One comment, besides great job <3

File: frontend/src/hooks/Group/useGroupScreenData.tsx (hunk 1 above)
Comment (AnCichocka) [reply frontend/src/hooks/Group/useGroupScreenData.tsx:67]: you're right, I missed that, thanks! <3

**Review by Szyntos — APPROVED** [review]
yaaay, looks god

**Pull request merged**